templates/index.html

```diff
@@ -1,27 +1,31 @@
 {% extends 'base.html' %}
 
 {% block title %}
-    Očkování
+Očkování
 {% endblock %}
 
 {% block head %}
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/styles.css') }}"/>
+<link rel="stylesheet" href="{{ url_for('static', filename='css/styles.css') }}"/>
 {% endblock %}
 
 {% block body %}
-    <h2>Aplikace s přehledem volných míst na COVID-19 očkování</h2>
+<h2>Aplikace s přehledem volných míst na COVID-19 očkování</h2>
 
-    {% if login %}
-        <h3>Logged as: {{ login }} </h3>
-    {% endif %}
-    <p>Povídání o aplikaci a co umí.</p>
+<p>Cílem této aplikace je nabídnout rychlý přehled o volných kapacitách v očkovacích centrech. Aktuální aplikace, které
+    existují, tuto funkcionalitu rychlého přehledu nemají a proto jsme tu my. Aktualizace jsou momentálně manuální,
```

Comment:
to trochu působí, že to tam vyplňujeme ručně :-)

Reply:
poloautomatické?

Reply:
To zmenime brzy, zitra / pozitra.